### PR TITLE
Allow ILC and Expert Mode to select mW-vs-PWM

### PR DESCRIPTION
**Pull request by mzieg**
- addresses #159 
- also refactored LaserPowerControl with new ctl architecture

### CODE REVIEW

**Comment by mzieg** [comment]
Moved WhatsThis button and rendered WhatsThis for new comboBox

File: enlighten/assets/uic_qrc/enlighten_layout.ui [hunk 1]
```diff
@@ -15066,7 +15066,7 @@ p, li { white-space: pre-wrap; }
...
...gt;In general, all spectrometers with multi-mode lasers (&lt;b&gt;MML&lt;/b&gt;) receive laser power calibrations, and are therefore recommended for control through mW. Spectrometers with single-mode lasers (&lt;b&gt;SML&lt;/b&gt;) typically do not receive laser power calibrations, and are therefore recommended for control through pulse-modulation width (&lt;b&gt;PWM&lt;/b&gt;) duty-cycle, i.e. a rough percentage of full power.&lt;/p&gt;&lt;p&gt;Therefore, most MML and SML spectrometers will not see this option, and will not be given the option to configure how laser power is controlled.&lt;/p&gt;&lt;p&gt;The special case is for Integrated Laser with Interchangeable Coupling (&lt;b&gt;ILC&lt;/b&gt;) models, which have an MML exposed through a fiber and probe.  Although these units will receive a laser power calibration in the factory, and therefore are able to set power in mW, the user may elect to use a different fiber and probe than was used to generate the calibration.&lt;/p&gt;&lt;p&gt;In such cases the calibration, although present, is invalid, and we need to give the user the ability to control the laser through percentage as well.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (samiebee43) [inline enlighten/assets/uic_qrc/enlighten_layout.ui:15069]: Single character change `all spectometers with multi-mode lasers (&lt;b& <---`

Comment (samiebee43) [reply enlighten/assets/uic_qrc/enlighten_layout.ui:15069]: before: `(&lt;bgt;MML&lt;/b&gt;)`
rendering: `(<bgt;MML</b>)` incorrect XML

after: `(&lt;b&gt;MML&lt;/b&gt;)`
rendering: `(<b>MML</b>)` correct XML

OK this change is good, just needed some writing space to understand it

File: enlighten/Controller.py [hunk 2]
```diff
@@ -1095,6 +1096,16 @@ def help_callback(self):
         log.debug(f"opening {url}")
         webbrowser.open(url)
 
+    def whats_this_callback(self):
```
Comment (samiebee43) [inline enlighten/Controller.py:1099]: This was moved from Measurements. Good idea.

**Comment by samiebee43** [comment]
Where was it located before?

**Comment by mzieg** [comment]
> Where was it located before?

Sorry, can you expand on that?

**Comment by samiebee43** [comment]
> Sorry, can you expand on that?

image

Referring to the original location of the button.

**Comment by mzieg** [comment]
Ah.  It had been atop the Measurements clipboard, because it initially only affected Thumbnails (was introduced as a replacement for Thumbnail tooltips).

**Pull request merged**